.NET/Microsoft.Recognizers.Text.DateTime/Arabic/Parsers/ArabicSetParserConfiguration.cs

```diff
@@ -38,6 +39,8 @@ public class ArabicSetParserConfiguration : BaseDateTimeOptionsConfiguration, IS
         private static readonly Regex YearTypeRegex =
             new Regex(DateTimeDefinitions.YearTypeRegex, RegexFlags);
 
+        private static readonly List<string> FutureTerms = (List<string>)DateTimeDefinitions.FutureTerms;
```

Comment:
Didn't we agree not to call it FutureTerms? Well, this is not a blocker to merge. But if there are other fixes to do, please rename this and the corresponding method.

Reply:
renamed in every file.